### ISSUE TITLE
Form Autofocus is bad for accessibility

### ISSUE THREAD

**Issue by tiagofassoni** [pennersr/django-allauth#777]
For instance, on the Login template, when using a screenreader, I should first hear the title, then the description, the list of available accounts and finally the login form.

Instead, because of the autofocus, I hear "username or e-mail first". Also, because screenreaders work using element focus, instead of navigating from top to bottom, I have to navigate from _middle_ to top, which violates the usability principle of "least surprise".

Google's homepage uses autofocus (with ARIA labels for screenreaders), but Google's homepage has one absolutely unambiguous point of attention, which the user will look as soon as the page loads: search. 

In constrast, allauth login page has several points of attention, like all the social accounts it allows login with.

(Trivia: I discovered this by asking a blind lawyer to navigate a website in which I used django-allauth. He found himself lost at the login page because of the autofocus)

**Comment by heliodor** [comment]
The login form uses autofocus whereas the signup form does not. Was this intentional?

**Comment by 9mido** [comment]
You can use https://pypi.org/project/django-widget-tweaks/ and override the templates to temporarily fix this by removing the autofocus attribute. This is a temporary fix though since it is not built into allauth.

**Comment by pennersr** [comment]
Closed via b9ca281c

**Comment by pennersr** [comment]
https://codeberg.org/allauth/django-allauth/issues/777
